kong/ids_test.go

```diff
@@ -0,0 +1,129 @@
+package kong_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kong/go-kong/kong"
+)
+
+type idFillable interface {
+	FillID() error
+}
```

Comment:
Oh I missed this one !
Can't you make idFillable public in kong/ids.go ? 
This way you can use it here with kong.IDFillable

Reply:
Yeah, my point was to just reduce public API to a minimum, but I guess exporting an interface won't hurt us.

Reply:
https://github.com/Kong/go-kong/pull/299/commits/ce04fc3eb240e1abaf3f065592feac6887d36917